config/checkstyle/tree_walker_module_directives.xml

```diff
@@ -67,14 +67,14 @@
     <property name="format" value="\bnCopies\b"/>
     <property name="message" value="Usage of the prohibited Java Collections.nCopies"/>
 </module>
-        <!-- io.vavr.collection.Traversable#peek and java.util.stream.Stream#peek
-                unexpectedly differ in their behaviour.
-                To avoid possible bugs, invocation of any method called "peek" is forbidden. -->
+<!-- io.vavr.collection.Traversable#peek and java.util.stream.Stream#peek
+    unexpectedly differ in their behaviour.
```

Comment:
Since you bumped against this yourself... pls leave a comment on what's the problem specifically with Vavr List's `peek`.